### DISCUSSION TITLE
这里就是绳网吗

### DISCUSSION THREAD

**Discussion by Yitaruose-BaiXu** [share121/inter-knot#1846] [其他]
![eae9bf02006110c6b32cd1d4689b0335](https://github.com/user-attachments/assets/7642943c-0912-4925-9023-df0d9cb4ae1c)
好新奇的感觉

**Comment by hlje233** [comment]
确实

**Comment by adrianlao647** [comment]
刚刚才来

**Comment by HaChi0504** [comment]
超神奇我也剛來

**Comment by wdf-86** [comment]
确实感觉挺新奇的，我也是刚刚来

**Comment by icenow188** [comment]
加一，搞了两个小时终于进来了

**Comment by haifanhen6** [comment]
_评论区那个说的挺清楚的，十分钟就搞定了_

**Comment by skysnown** [comment]
达 达 利 亚 ！！！

**Reply by HEYD66** [reply]
我也刚来

**Comment by yileinafs** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/bfaa94e859e32501b1ffcf662fcef4be_6155242370256719352.png" alt="机巧鸟 尖叫" width="75" >
